crawler/crawler/saveToElasticsearch.go

```diff
@@ -122,4 +124,31 @@ func (repo *Repository) generateSlug() string {
 	return fmt.Sprintf("%s-%s", repo.Pa.CodiceIPA, vendorAndName)
 }
 
+// DeleteByQueryFromES delete record from elasticsearch
+// that will match search sting for publiccode.url field
+func (c *Crawler) DeleteByQueryFromES(search string) error {
+	// Search with a term query
+	termQuery := elastic.NewTermQuery("publiccode.url", search)
 
+	// Put publiccode data in ES.
+	ctx := context.Background()
+	searchResult, err := c.es.DeleteByQuery().
+		Index(c.index).
+		Type("software").
+		Query(termQuery). // specify the query
+		Do(ctx)           // execute
+	if err != nil {
+		return err
+	}
+
+	if searchResult == nil {
+		return errors.New("Generic error on DeleteByQueryFromES()")
+	}
+
+	if searchResult.Deleted == 0 {
+		return errors.New("No records deleted for searched query")
+	}
+
+	log.Infof("Deleted %d record from ES", searchResult.Deleted)
```

Comment:
We have the `search` string, I'd like to see also that in the logs, what do you think? I guess it's better to save it at this stage because we have both info before returning.
```suggestion
	log.Infof("Deleted %d record from ES linked to %s", searchResult.Deleted, search)
```

Reply:
`delete` is generally a CLI command, but of course, in future implementation could be useful log that information.